### DISCUSSION TITLE
Can't find qt building dependencies

### DISCUSSION THREAD

**Discussion by sekatol** [audacity/audacity#6275] [Development]
Where is the building requirements of qt branch? I installed the requirements in BUILDING.md, but it says  missing other requirements when building.

**Comment by sekatol** [comment]
In BUILDING.md:
```
Linux
We use GCC 9 and later, but any C++17 compliant compiler should work.

Here are the dependencies you need to install on various distribution families.

Debian and Ubuntu
$ sudo apt-get update
$ sudo apt-get install -y build-essential cmake git python3-pip
$ sudo pip3 install conan
$ sudo apt-get install libgtk2.0-dev libasound2-dev libjack-jackd2-dev uuid-dev
```
I installed these dependencies.

**Reply by sekatol** [reply]
But still missing dependencies, like "QString"

**Comment by LWinterberg** [comment]
The Qt branch isn't functional, it merely has some UI shell. I'd recommend against tinkering with it at the moment.

If you really want to get it going, following what our CI does probably is a good start.
